test/testpy/__init__.py

```diff
@@ -89,6 +89,13 @@ def GetCommand(self):
 
     return result
 
+    # We want to use the provided openssl.cnf, not the default one of 
+    # the shared lib
+    has_shared_openssl = Execute([vm,
+       "-p", "process.config.variables.node_shared_openssl"], context)
+    if has_shared_openssl.stdout.rstrip() == "true":
+       os.environ['OPENSSL_CONF'] = "./deps/openssl/openssl/apps/openssl.cnf"
```

Comment:
cc @nodejs/python I'm not sure if this would be executed (or is even a syntax error) as it's outside a method in a class definition.

Reply:
@richardlau It’s unconventional, especially in the sense that it is used to modify global state and the static class variable is not used otherwise, but it should work. But yeah, there’s probably a better place for this.

Reply:
At a minimum it is _unreachable code_ because the `return` statement that precedes it is not optional.  My bet is that pylint and perhaps mypy and flake8 would flag it.